.github/pull_request_template.md

```diff
@@ -22,6 +22,7 @@
 - [ ] Documentation (README, local.env.dist, etc.)
 - [ ] Unit tests created or updated
 - [ ] Run `make composershow`
+- [ ] Run `make psr2`
```

Comment:
Probably doesn't matter right now as only 2 commands, but could make an additional make command to run all these rather than having to do `make composershow psr2 ...etc`

Reply:
Will consider it for the future